packages/rum-core/src/domain/tracing/tracer.spec.ts

```diff
@@ -522,6 +567,53 @@ describe('tracer', () => {
       expect(context.init!.headers).not.toContain(jasmine.arrayContaining(['x-datadog-trace-id']))
       expect(context.init!.headers).not.toContain(jasmine.arrayContaining(['X-B3-TraceId']))
     })
+    it('should not add headers when trace not sampled and config set to sample', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏typo
```suggestion
    it('should not add headers when trace not sampled and config set to sampled', () => {
```